instrumentation/nocode/src/main/java/com/splunk/opentelemetry/instrumentation/nocode/YamlParser.java

```diff
@@ -33,7 +33,7 @@
 
 public final class YamlParser {
   private static final Logger logger = Logger.getLogger(YamlParser.class.getName());
-  // FIXME support method override selection - e.g., with classfile method signature or something
+  // FUTURE support method override selection - e.g., with classfile method signature or something
```

Comment:
One issue with instrumenting methods from sub classes is that we currently can't easily tell from advice code which rule is tied to the method invocation because we can't just look up the rule based on the invoked class and method name. I haven't tried it out yet, but I believe we could make this work by adding an api in the upstream that would let us pass extra data, such as rule id, into the invoked advice method.